### PR TITLE
[ADDED]Getting Started

### PR DESCRIPTION
**Pull request by tanishakumar18**
Added Getting started for TNP Website

### CODE REVIEW

**Comment by kamalika0363** [comment]
[ADDED] #78   
README for TnP Website

File: Getting started.md
Comment (vish-rt) [inline Getting started.md:1]: The content of this file should be in `README.md` only. Putting this in any other file would not reflect within the repository's description.
- [ ] Copy everything written in this file
- [ ] Replace the contents of `README.md` with this

File: Getting started.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+**Info:**
```
Comment (vish-rt) [inline Getting started.md]: While writing a project's description, always begin with the project's name. Leverage heading tags (#) to highlight them.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Recommend having a look at the suggestions and make new commits to your fork.

**Review by vish-rt — APPROVED** [review]
Great!

**Pull request merged**